src/main/java/co/edu/itc/progrmacion/AppConfig.java

```diff
@@ -1,11 +1,44 @@
 package co.edu.itc.progrmacion;
 
-import org.springframework.context.annotation.ComponentScan;
-import org.springframework.context.annotation.Configuration;
-import org.springframework.context.annotation.PropertySource;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.*;
+import org.springframework.context.support.PropertySourcesPlaceholderConfigurer;
+import org.springframework.data.jdbc.repository.config.EnableJdbcRepositories;
+import org.springframework.jdbc.datasource.DriverManagerDataSource;
+
+import javax.sql.DataSource;
 
 @Configuration
 @ComponentScan("co.edu.itc.progrmacion")
+@EnableJdbcRepositories("co.edu.itc.progrmacion")
 @PropertySource("classpath:application.properties")
 public class AppConfig {
```

Comment:
Debes extender de AbstractJdbcConfiguration para poder trabajar con solo spring framework (Realizar los imports respectivos)

```suggestion
public class AppConfig extends AbstractJdbcConfiguration {
```